specs/lightclient/beacon-chain.md

```diff
@@ -192,21 +187,19 @@ class BeaconState(Container):
     # Slashings
     slashings: Vector[Gwei, EPOCHS_PER_SLASHINGS_VECTOR]  # Per-epoch sums of slashed effective balances
     # Participation
-    previous_epoch_participation: List[ParticipationFlags, VALIDATOR_REGISTRY_LIMIT]
-    current_epoch_participation: List[ParticipationFlags, VALIDATOR_REGISTRY_LIMIT]
+    previous_epoch_participation: List[ParticipationFlags, VALIDATOR_REGISTRY_LIMIT]  # [New in HF1]
+    current_epoch_participation: List[ParticipationFlags, VALIDATOR_REGISTRY_LIMIT]  # [New in HF1]
     # Finality
     justification_bits: Bitvector[JUSTIFICATION_BITS_LENGTH]  # Bit set for every recent justified epoch
     previous_justified_checkpoint: Checkpoint
     current_justified_checkpoint: Checkpoint
     finalized_checkpoint: Checkpoint
-    # Light client sync committees
-    current_sync_committee: SyncCommittee
-    next_sync_committee: SyncCommittee
-    # How many inactivity leak epochs have there been in the previous penalty period?
-    leak_epoch_counter: uint64
-    # Increases when the validator misses epochs in an inactivity leak, decreases when the validator
-    # is online in an inactivity leak, inactivity leak penalties are proportional to this value
-    leak_score: List[uint64, VALIDATOR_REGISTRY_LIMIT]
+    # Leak
+    leak_epochs_counter: Epoch  # [New in HF1]
```

Comment:
The definition of custom type `Epoch` is "which epoch", not "epoch count". So it should be `uint64`.
```suggestion
    leak_epochs_counter: uint64  # [New in HF1]
```

Reply:
agreed